### ISSUE TITLE
Improve the smaller scale clustering of quasars auto-correlation

### ISSUE THREAD

**Issue by londumas** [igmhub/LyaCoLoRe#31]
Good jobs London-team, these results are very good.

Running the auto-correlation of quasars on the full sample gives the following results.
The parameters are very good: the three cosmological parameter and the bias of quasars:
https://github.com/igmhub/desi_QA/blob/master/Mocks/QA_autoQSO.ipynb.

```
ap = 1.002 +/- 0.030
at = 1.009 +/- 0.019
beta_q = 0.258 +/- 0.021
f = 0.967 +/- 0.064
```

The correlation function is good down to ~ 40 Mpc/h.
I was wondering if:
- we have the will and the time to try to improve it down to smaller scales
- we have the ways to do so

![autoqso_desi_mocks_wedgeall](https://user-images.githubusercontent.com/3166436/48452464-78903100-e76c-11e8-9322-c82277fa4783.png)

**Comment by fjaviersanchez** [comment]
@andreufont @londumas: we can try to use 2LPT instead of the lognormal box so the low scale behavior is more realistic. However, the switch still doesn't guarantee to have realistic auto-correlations at scales smaller than 20 Mpc/h and the usage of 2LPT will create a different set of problems. I'd vote to keep this as a secondary goal for now (more so given the limited manpower that we have). Having said so, I am happy to generate a box with QSOs only and the same cosmology with 2LPT if you'd like to check it out @londumas and see if that helps.

**Comment by londumas** [comment]
@fjaviersanchez, Thank you for the idea. I don't have much time for now and will have even less in the future. However I wanted to give you guys the current state of this measurement. Would it take you a long time to generate "a box with QSOs only and the same cosmology with 2LPT"? If not too much time, let's do it. If it is too much work let's forget it.

I forgot to give you guys one of the reason I care about this: we are starting with @TEtourneau a project of measuring f*sigma_8 at z>2 with auto_lya combined with cross_qso_lya. Adding auto_qso would help the measurement and better understand the systematics.

**Comment by fjaviersanchez** [comment]
Thanks @londumas! This should not take a lot of time (this should be done in 30 minutes or so if I find the time) the problem is that I am currently pretty overbooked. I'll try to find some time today, if I can't make it, I'll write down the instructions on how to do it and maybe we can find somebody else to work on this.

**Comment by londumas** [comment]
@fjaviersanchez, No need to do it today. We have time to look for this. Thank you very much.

**Comment by andreufont** [comment]
@londumas - Note that if the auto-correlation of quasars is off at r<40 Mpc/h, the cross-correlation will also be off to some degree.

And even thought it might be ineteresting to run 2LPT, we would not be able to run it with the same resolution, we would need to go to 5 Mpc/h cells, what might open other cans of worms. Moreover, the current code by James to add extra power assumes that the input field is Gaussian, not sure how easy it would be to use 2LPT density fields.

**Comment by fjaviersanchez** [comment]
@andreufont I agree with your assessment. However, I think that it would still be interesting to make a proof-of-concept study (see if just the QSOs by themselves have the correct low-scale power). That way we can quickly decide whether this might be something worth looking at in the future or, if we want more realistic low-scale correlation functions, we have to change the paradigm completely. What do you think?
